### PR TITLE
New thump and full image

### PR DESCRIPTION
**Pull request by TheMBeat**
Fix  [#415](https://github.com/nextcloud/cookbook/issues/414)

### CODE REVIEW

**Review by christianlupus — COMMENTED** [review]
Why are two different SVG files required? The idea was to have a single well-scalable image.

Additionally, it might be a good idea to popp things up with CSS+Vue changes so that the existing images are replaced by an SVG image + gray background of appropriate size.

**Comment by TheMBeat** [comment]
One SVG makes more Sense. I hadn't thought through to the end.  My mistake.  I'll improve it.

**Comment by christianlupus** [comment]
I will have a look later as I am quite busy currently due to day-time job.

File: src/components/RecipeImages.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <div v-if="$store.state.recipe.image" :class="{ 'collapsed': collapsed, 'printable': $store.state.recipe.printImage }">
+    <div v-if="$store.state.recipe.image" :class="{ 'collapsed': collapsed , 'printable': $store.state.recipe.printImage }">
```
Comment (christianlupus) [inline src/components/RecipeImages.vue:2]: This is a white-space only change. Is this really needed?

File: src/components/RecipeImages.vue [hunk 2]
```diff
@@ -24,17 +24,20 @@ export default {
...
-	    text-align: center;
-	    margin-bottom: 1rem;
-	}
+    div {
+        display: block;
+        width: 100%;
+        text-align: center;
+        margin-bottom: 1rem;
+
+    }
```
Comment (christianlupus) [inline src/components/RecipeImages.vue:33]: What are these changes? I do not get the difference here.

File: src/components/RecipeImages.vue [hunk 3]
```diff
@@ -24,17 +24,20 @@ export default {
...
     img {
         cursor: pointer;
         width: 100%;
         max-width: 950px;
+        background-color: #bebdbd;
     }
+
```
Comment (christianlupus) [inline src/components/RecipeImages.vue:40]: White-space only can be dropped.

**Review by christianlupus — CHANGES_REQUESTED** [review]
There is a bug in the PR. When opening the cookbook app, I get these errors in the console:
Screenshot_20201206_170045

**Comment by TheMBeat** [comment]
I look at this tomorrow.

**Review by TheMBeat — COMMENTED** [review]
@christianlupus are the changes ok for you now?

**Review by christianlupus — APPROVED** [review]
Looks good and I see no issues here.

**Comment by Teifun2** [comment]
This leads to a lot of problems in my App development.

The browser is totally happy if instead of a PNG he receives a SVG. But for normal App Components this always results in crashes deep in the library. As it gets a proper response but cannot parse the response into an image of any type.


Would it be ok if i changed the code so that the RecipeController returns 404 if no image is found. And the frontend itself adds the SVG in these cases? This seems a much cleaner "API" solution.

**Comment by christianlupus** [comment]
@Teifun2 if this needs more discussion, we might be better of in a new issue instead of this PR.

Apart from that, can't you check the `Content-Type` header before?

**Pull request merged**